salt/fileserver/__init__.py

```diff
@@ -272,6 +272,34 @@ def is_file_ignored(opts, fname):
     return False
 
 
+def clear_lock(clear_func, lock_type, remote=None):
+    '''
+    Function to allow non-fileserver functions to clear update locks
```

Comment:
This function is not exposed to the docs and can't be executed directly from the CLI, so I don't see a benefit for adding one.